cdk/checkfiles_runner/lambdas/wait_checkfiles/main.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+
+import boto3
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    force=True
+)
+
+
+class CommandInProgress(Exception):
+    pass
+
+
+def wait_checkfiles_command_to_finish(event, context):
+    ssm = boto3.client('ssm')
+    instance_id = event['instance_id']
+    command_id = event['command_id']
+
+    result = ssm.get_command_invocation(
+        CommandId=command_id,
```

Comment:
What's the command? How is it checking?

Reply:
Command is in the previous lambda here: https://github.com/IGVF-DACC/checkfiles/blob/CHECK-51-automatic-run/cdk/checkfiles_runner/lambdas/run_checkfiles/main.py#L46 and the command id is passed through.